controls/V-41808.rb

```diff
@@ -12,30 +14,42 @@
 cause an attacker to take a large amount of time and resources to guess,
 reducing the likelihood of an attacker guessing a session ID.
   "
+  
+  desc  "check", "Review the NGINX web server documentation and deployed configuration 
+  to see how long the generated session identifiers are.
+
+  If it is determined that the web server is not required to perform session 
+  management, this check is Not Applicable. 
+
+  NGINX web server versions after 1.11.0 have the $request_id embedded variable 
```

Comment:
We should also use the `nginx_version` input here as well so the text updates automatically as well